nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -97,9 +108,9 @@ def __init__(
         if activation in ['geglu', 'reglu', 'swiglu']:
             # Separate linear layer for *GLU activations.
             # Source: https://github.com/huggingface/transformers/blob/bee361c6f1f7704f8c688895f2f86f6e5ff84727/src/transformers/models/t5/modeling_t5.py#L292
-            self.dense_h_to_4h_2 = tensor_parallel.ColumnParallelLinear(
+            self.dense_h_to_4h_2 = ColumnLinear(
                 hidden_size,
-                ffn_hidden_size,  # NOTE: When using *glu, divide ffn dim by 2/3 to keep overall params the same.
+                ffn_hidden_size,  # NOTE: When using geglu, divide ffn dim by 2/3 to keep overall params the same.
```

Comment:
Replaced during one of the merges, I think this should be for the functions *glu like reglu too, instead of only geglu. Correct me if I am wrong

Reply:
You're right, missed it. Will clean it up.